CMakeLists.txt

```diff
@@ -173,6 +174,21 @@ else()
   endif()
 endif()
 
+include(CheckCCompilerFlag)
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le")
+  message(STATUS " we are ppc64le")
+  CHECK_C_COMPILER_FLAG("-maltivec" HAS_ALTIVEC)
+  if(HAS_ALTIVEC)
+    message(STATUS " HAS_ALTIVEC yes")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -maltivec")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -maltivec")
```

Comment:
nit, this might break the intention of the `PORTABLE` option. if a building host is targeting ppc64le and supports altivec instructions, but the CPU on deployed machine is ppc64le but does not supports altivec?

if we can always assume that altivec is available as long as it's ppc64le, we are good.

Reply:
Yes, it is safe to assume all ppc64le systems support altivec.  (The reverse is not true.  In other words, altivec support was added prior to Power8.  In the Power8 timeframe the ppc64 focus for all Linux distros is little endian, but for backward compatibility reasons I don't want to break anything running on Power7 or earlier hardware, or running in big endian mode).